test/fixtures/basic.mdx

```diff
@@ -8,8 +8,6 @@ export const meta={
 
 Lorem ipsum dolor sit amet, consectetur adipiscing **elit**. Ut ac lobortis <b>velit</b>.
 
-<!-- This is a comment -->
```

Comment:
should this perhaps be kept as `{/* This is a comment */}`

Reply:
Right.